segmentation/utils/spatial_analysis_utils.py

```diff
@@ -1,16 +1,17 @@
 import numpy as np
-import math
+import pandas as pd
 import skimage.measure
 from scipy.spatial.distance import cdist
 
 
 def calc_dist_matrix(label_map):
     """Generate matrix of distances between center of pairs of cells
```

Comment:
Let's modify this to instead take in an array of label maps, loop over them, and return the output as an array of dist_mats. The input to this function is the segmentation_labels.xr produced by the segmentation function: https://github.com/angelolab/segmentation/blob/76fbf4682f108bd047f878262337c0be9a3c26e5/segmentation/utils/segmentation_utils.py#L70. It has a shape of [points, rows, cols, 1]